README.md

```diff
@@ -464,7 +464,8 @@ This API influences the state of the [Application Under Test](#application-under
 
 Name | Type | Required | Description | Example
 --- | --- | --- | --- | ---
-bundleId | string | yes | bundle identifier of the app to be launched or activated | com.apple.TextEdit
+bundleId | string | no | Bundle identifier of the app to be launched or activated. Either this argument or the `path` one must be provided | com.apple.TextEdit
+path | string | no | Full path to the app bundle. Either this argument or the `bundleId` one must be provided | /Applications/Xcode.app
```

Comment:
why not use the `app` capability for this so it is semantically equivalent to the other drivers?